Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs

```diff
@@ -0,0 +1,314 @@
+        private static string changelogDescription = "The Change Log is a list of changes made to MirrorNG. Sometimes these changes can cause your game to break.";
+        private static string quickStartDescription = "The Quick Start Guide is meant for people who just started using MirrorNG. The Quick Start Guide will help new users learn how to accomplish important tasks. It is highly recommended that you complete the guide.";
+        private static string bestPracticesDescription = "This page describes the best practices that you should use during development. Currently a work in progress.";
+        private static string faqDescription = "The FAQ page holds commonly asked questions. Currently, the FAQ page contains answers to: \n\n   1. Syncing custom data types \n   2. How to connect \n   3. Host migration \n   4. Server lists and matchmaking";
+        private static string sponsorDescription = "Sponsoring will give you access to Mirror PRO which gives you special access to tools and priority support.";
+
+        //titles of the redirect buttons
+        private static string welcomePageButtonTitle = "Visit API Reference";
+        private static string changelogPageButtonTitle = "Visit Change Log";
+        private static string quickStartPageButtonTitle = "Visit Quick Start Guide";
+        private static string bestPracticesPageButtonTitle = "Visit Best Practices Page";
+        private static string faqPageButtonTitle = "Visit FAQ";
+        private static string sponsorPageButtonTitle = "Sponsor Us";
+
+        #endregion
+
+        #region Urls
+
+        private static string welcomePageUrl = "https://mirrorng.github.io/MirrorNG/";
+        private static string quickStartUrl = "https://mirrorng.github.io/MirrorNG/Articles/Guides/CommunityGuides/MirrorQuickStartGuide/index.html";
+        private static string changelogUrl = "https://github.com/MirrorNG/MirrorNG/commits/master";
+        private static string bestPracticesUrl = "https://mirrorng.github.io/MirrorNG/Articles/Guides/BestPractices.html";
+        private static string faqUrl = "https://mirrorng.github.io/MirrorNG/Articles/Guides/FAQ.html";
+        private static string sponsorUrl = "";
+        private static string discordInviteUrl = "https://discord.gg/N9QVxbM";
+
+        #endregion
+
+        //window size of the welcome screen
+        private static Vector2 windowSize = new Vector2(500, 600);
+
+        //returns the path to the Mirror folder (ex. Assets/Mirror)
+        private static string MirrorPath
+        {
+            get
+            {
+                //get an array of results based on the search
+                string[] results = AssetDatabase.FindAssets("", new[] { "Assets" });
```

Comment:
I don't think this works with UPM.

A better way is to make this property not static,  call `MonoScript.FromScriptableObject(this)` to get the script asset and then call `AssetDatabase.GetAssetPath()` to get the path to this script.